============================= test session starts ==============================
platform darwin -- Python 3.9.6, pytest-8.1.1, pluggy-1.4.0
rootdir: /Users/cohithereweer/Desktop/formatFlick-engine
collected 4 items

tests/test_engine/test_cEngine.py ...                                    [ 75%]
tests/test_formatflick.py F                                              [100%]

=================================== FAILURES ===================================
__________________________________ test_core ___________________________________

    def test_core():
        source = os.path.join(os.getcwd(), "tests", "sample_files", "sample-1.csv")
        with pytest.raises(Exception) as err:
            obj = ff.formatflick(
                source=source,
                destination=None,
                destination_extension=None,
                verbosity=None,
                mode='file'
            )
>       assert str(err) == "destination and destination_extension both cannot be None."
E       assert "<ExceptionIn...ts') tblen=3>" == 'destination ...nnot be None.'
E         
E         - destination and destination_extension both cannot be None.
E         + <ExceptionInfo Exception('The source path dont exits') tblen=3>

tests/test_formatflick.py:19: AssertionError
=========================== short test summary info ============================
FAILED tests/test_formatflick.py::test_core - assert "<ExceptionIn...ts') tbl...
========================= 1 failed, 3 passed in 0.58s ==========================
